examples/getAllAccounts.js

```diff
@@ -0,0 +1,29 @@
+const { getAllAccounts, Account, Vault } = require('..');
+const { loadStdlib } = require('@reach-sh/stdlib');
+
+(async () => {
+  const account = new Account({
+    mnemonic: 'lens sell....',
+    network: 'LocalHost',
+  });
+  const VAULT_ID = 123;
+  const vault = new Vault({ id: VAULT_ID });
+  const reach = loadStdlib('ALGO');
+  const indexer = new reach.algosdk.Indexer(
+      process.env.REACT_APP_INDEXER_KEY,
+      process.env.REACT_APP_INDEXER_URL,
+      process.env.REACT_APP_INDEXER_PORT,
+  );
```

Comment:
This is perfect for the example, but am thinking in the future, the internal implementation for this can be modified so people do not need to instantiate the indexer or have the algosdk as a dependency and simply pass in strings to the SDK and the SDK takes care of everything on their behalf. This might also help with breaking changes compatibility and the likes.

Reply:
For now I might even hard code in the example the algoindexer values we use. I agree though; it'd be great if our SDK intitialised the indexer.